pavelib/assets.py

```diff
@@ -597,6 +607,19 @@ def process_npm_assets():
     """
     Process vendor libraries installed via NPM.
     """
+    def copy_vendor_library(library, skip_if_missing=False):
```

Comment:
Here's the ticket for improving the way we consume NPM assets:

https://openedx.atlassian.net/browse/TNL-6001